Dev milestone 3

mistermaxis-zz opened the pull request:
### Changes Implemented:

- Added: "In the magazine" and "Editor’s Pick" section.

#### Newspaper main sections:
- Added US section.
- Added World section.
- Added Business section.
- Added Tech & Science section.
- Added Culture section.
- Added Sports section.
- Added Expert forum section.

mistermaxis-zz committed:
Install and setup linter

mistermaxis-zz committed:
Add images and icons for header

mistermaxis-zz committed:
Add elements for banner part of header

mistermaxis-zz committed:
Add style.css with dummy class

mistermaxis-zz committed:
Add elements for nav part of header.

mistermaxis-zz committed:
Add bootstrap CDN

mistermaxis-zz committed:
Style header and fix linter errors

mistermaxis-zz committed:
Change to header element

mistermaxis-zz committed:
Delete package-lock.json

mistermaxis-zz committed:
Merge branch 'footer' into dev-milestone-1

mistermaxis-zz committed:
Merge branch 'master' into dev-milestone-1

mistermaxis-zz committed:
Deleted hint report html file

mistermaxis-zz committed:
Remove section from header and change date to span

mistermaxis-zz committed:
remove link to stylesheet.css

mistermaxis-zz committed:
Add hint-report to .gitignore

mistermaxis-zz committed:
Add style for footer in style.css

Anas-jaf committed:
added the first culomn and second column but i need to add latest news to it too

IOT-club-mutah committed:
finished the second mileston

mistermaxis-zz committed:
Update index.html

Remove stray "i" tag in line 460 index.html

mistermaxis-zz committed:
Add images for "In Magazine" section

mistermaxis-zz committed:
Add  cover and first article

- Add cover and first article for "in magazine" section.
- Add style using bootstrap classes.

mistermaxis-zz committed:
Update style.css

Add class to underline card text in red for the "In Magazine" section.

mistermaxis-zz committed:
Update index.html

Add grid layout to elements in "In Magazine" section.

mistermaxis-zz committed:
Create joey.jpg

Add image for "Editor's Pick" section.

mistermaxis-zz committed:
Update style.css

Rename: 'mag-card-link' -> 'card-link'

mistermaxis-zz committed:
Update index.html

-Add "Editor's Pick" section.
-Add first article in the section.
-Add style for first article in the section.

mistermaxis-zz committed:
Update index.html

Add grid layout for different sizes in "Editor's Pick" section.

mistermaxis-zz committed:
Update index.html

Correct "src" attribute in img elements so the image file loads properly.

IOT-club-mutah committed:
added the newsletter and subscribe section

IOT-club-mutah committed:
Merge branch 'dev-milestone-3' of https://github.com/mistermaxis/bootstrap-project into dev-milestone-3

IOT-club-mutah committed:
fixed the merge issue in the html page

mistermaxis-zz committed:
Update index.html

Remove slash at the beginning of src attribute in image elements.

mistermaxis-zz committed:
Update style.css

Change rule selector to update change in html: 'i' -> 'a'.

mistermaxis-zz committed:
Create logo-newsweek-red.svg

Add logo as svg file instead of svg element.

mistermaxis-zz committed:
Update index.html

Change layout to have the container bootstrap element contain all the content

mistermaxis-zz committed:
Update README.md

mistermaxis-zz committed:
Create website-clone-screenshot.png

Add website screenshot for readme.

mistermaxis-zz committed:
Update README.md

Add live demo link and screenshot.

mistermaxis-zz committed:
Update README.md

Change format for screenshot link.

mistermaxis-zz committed:
Update README.md

mistermaxis-zz committed:
Update README.md

Add details to and screenshot to README.md file.

mistermaxis-zz committed:
Update index.html

Fix webhint errors.

voscarmv reviewed (changes requested):
# Nice work so far. 👍 
# Impressive work. 💻 👀 ✨ 
- You've done an outstanding job with this project. Your page looks extremely close to the original. Congratulations. Your project is almost complete.
## However, there are some changes required 👀 🧰 🔨 🔧 
- The last section shows the following misalignment:
![image](https://user-images.githubusercontent.com/2739245/110885222-14ce9200-82ac-11eb-8163-f86648ca5b84.png)
- Please fix this to make it resemble the original:
![image](https://user-images.githubusercontent.com/2739245/110885485-8b6b8f80-82ac-11eb-95bf-3ede3c4743f3.png)
- Also please make sure to fix all your failing Github Action tests.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

mistermaxis-zz committed:
Update index.html

Fix webhint errors

mistermaxis-zz committed:
Update index.html

Fix webhint errors

mistermaxis-zz committed:
Update style.css

Move style for div with background to css rule in style.css.

mistermaxis-zz committed:
Update index.html

Modify layout in last section to have both elements align properly.

mistermaxis-zz committed:
Update style.css

Fix missing style for layout in last section.

mistermaxis-zz committed:
Update index.html

Add class to control height of elements in last section so they align horizontally.

mistermaxis commented:
### Changes Implemented:

- Changed layout in the last section of the page so that both elements align properly.

mistermaxis-zz committed:
Update index.html

Correct name of class in div element.

smcommits reviewed (approved):
# Status: Approved :+1:
## Hello, team! You have done a great job. I can see that you have implemented the changes requested by the previous TSE. This PR is now approved. You can go ahead and merge the branch. Congratulations and good luck!

mistermaxis-zz committed:
Update website-clone-screenshot.png

Update website screenshot
